src/main/java/frc/team166/robot/subsystems/Drive.java

```diff
@@ -131,4 +165,69 @@ protected void end() {
             }
         };
     }
+
+    public Command DriveDistance(double inches) {
+        return new SubsystemCommand("Drive Distance", this) {
+
+            double realDistanceInches = frontLidar.getDistance(true);
+
+            @Override
+            protected void initialize() {
+                drivePidController.setSetpoint(tempestGyro.getAngle());
+                drivePidController.enable();
+                drivePidController.reset();
+            }
+
+            @Override
+            protected void execute() {
+                m_drive.arcadeDrive(.2, angle);
+            }
+
+            @Override
+            protected boolean isFinished() {
+                if (realDistanceInches <= inches) {
+                    return true;
+                } else {
+                    return false;
+                }
+
+            }
+
+            @Override
+            protected void end() {
+                drivePidController.disable();
+            }
+        };
+    }
+
+    public Command setTurn(double degrees) {
```

Comment:
First off, should be upper camel (SetTurn), and secondly I think the name isn't quite complete... This does more than "set" a turn, it actually DOES it. Maybe call it "TurnByDegrees" or something?

Reply:
Functions should be lower camel, but the function name should still be changed to be more descriptive like Bill suggested